src/main/java/com/github/princesslana/eriscasper/gateway/commands/RequestGuildMembers.java

```diff
@@ -0,0 +1,21 @@
+package com.github.princesslana.eriscasper.gateway.commands;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.github.princesslana.eriscasper.data.Snowflake;
+import org.immutables.value.Value;
+
+/**
+ * @see <a href="https://discordapp.com/developers/docs/topics/gateway#request-guild-members">
+ *     https://discordapp.com/developers/docs/topics/gateway#request-guild-members</a>
+ */
+@Value.Immutable
+public interface RequestGuildMembers {
+  @JsonProperty("guild_id")
+  Snowflake getGuildId();
+
+  @JsonProperty("query")
```

Comment:
Be consistent with how we specify these `@JsonPropery` annotations. Either we always specify them, or we only specify them when we need to.

Reply:
Jackson by default converts by remove the `get` and lower casing the initial letter.

So, `getQuery` looks at the `query` field. `getGuildId` would look at the `guildId` field by default, which doesn't match with `guild_id` that we want.

Reply:
ah, alright, I'll go through and modify the json properties I set to match with that